### PR TITLE
Adds gorillas and cargorillas

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Adds gorillas and cargorillas

Gorilla:
- 220 HP
- 15-18 damage to mobs
- 20 damage to objects
- Can smash walls
- 0.5x speed
- 1.5x Burn, Toxin, Oxy damage
- Immune to clone and stamina damage
- Can carry one crate
- Can stand up on two legs if they want to
- Throws mobs they punch
- Can dismember unconscious mobs
- 1% chance of mutating from monkeys with over 1500 rads (balance figures subject to change)
- Hostile (they will find you and they will kill you)
- 1% chance of mutating when any carbon's `randmutb()` is called (cloning, genetics DNA modifiers, rad storms, and rarely by unstable mutagen)

Rampaging Gorilla:
- Red and runs at normal speeds
- Has 350 health
- Does more damage.
- Takes 1.25x Brute damage (Unlike the 1x parent)
- Takes 1x Burn and Oxy damage (Unlike the 1.5x parent)

Cargorilla:
- Can pick up 2 crates at the same time
- Has cargo access
- Pacifist
- Will eventually spawn in cargo in a subsequent PR

Added cargorilla crates, 250 credits. (mulebot crate cost)
Added gorilla crates, 1000 credits.

Added Magillitis bio-chip, turning you into rampaging gorilla when you use the action.

Added Magillitis bio-chip implanter to RD and Genetics traitor uplinks, 5TC.

Removed random crappy un-used big crates.

Added a bunch of mobs to the "jungle" faction so the gorilla will leave them alone, went with ones that matched thematically.

Moved simplemob var comments to autodoc, because that was annoying as hell.

## Why It's Good For The Game
Rampaging gorillas?

https://user-images.githubusercontent.com/12197162/230654007-a507e232-c74f-49f9-82e7-df9446ea61a2.mp4

## Images of changes
image

## Testing
Gorilla crate picking, punching things, etc.

## Changelog
:cl:
add: Added gorillas and cargorillas
add: Added Magillitis bio-chip implanter to RD and Genetics traitor uplinks, 5TC
/:cl:

### CODE REVIEW

**Comment by S34NW** [comment]
```
FAIL /datum/unit_test/emote 0.1s
	Emote /datum/emote/living/simple_animal/gorilla/ooga is usable, but not assigned a keybind.
```

guh

**Comment by Probablyjay19** [comment]
image

**Comment by SynthTwo** [comment]
gorillas :) gorilla friend

**Comment by Leanfrog** [comment]
https://www.youtube.com/shorts/yW8V--TrlJM

**Comment by S34NW** [comment]
> image

Added

**Comment by Probablyjay19** [comment]
> > image
> 
> 
> 
> Added

image

(Although, 5 TC is dirt fucking cheap. On TG it's 15 TC 🥶)

**Comment by S34NW** [comment]
> 
> (Although, 5 TC is dirt fucking cheap. On TG it's 15 TC 🥶)

True, but genetics is shit when it comes to traitor opportunities. And once you go gorilla, you can never go back.

**Comment by Denthamos** [comment]
> > (Although, 5 TC is dirt fucking cheap. On TG it's 15 TC 🥶)
> 
> True, but genetics is shit when it comes to traitor opportunities. And once you go gorilla, you can never go back.

There needs to be a reference when a geneticist turns themself into a gorilla, to the meme "Return to monke"

**Comment by S34NW** [comment]
```
[2023-04-07T11:11:02] Runtime in dna2_helpers.dm,244: Attempting to reset DNA from a missing head!
  proc name: head traits to dna (/datum/dna/proc/head_traits_to_dna)
  src: /datum/dna (/datum/dna)
  call stack:
  /datum/dna (/datum/dna): head_traits_to_dna
  /datum/dna (/datum/dna): ResetUIFrom
  /datum/dna (/datum/dna): ready_dna
  the monkey (872) (/mob/living/carbon/human/monkey): setup_dna
  the monkey (872) (/mob/living/carbon/human/monkey): Initialize
  the monkey (872) (/mob/living/carbon/human/monkey): Initialize
  Atoms (/datum/controller/subsystem/atoms): InitAtom
  Atoms (/datum/controller/subsystem/atoms): InitializeAtoms
  Atoms (/datum/controller/subsystem/atoms): Initialize
  Atoms (/datum/controller/subsystem/atoms): call_init
  Master (/datum/controller/master): Initialize```

Wasn't me, I swear

File: code/modules/mob/living/carbon/human/species/monkey.dm
Comment (Sirryan2002) [inline code/modules/mob/living/carbon/human/species/monkey.dm:46]:
```suggestion
```
dangling return

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm
Comment (Sirryan2002) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm:67]:
```suggestion
	gorilla.is_bipedal = !gorilla.is_bipedal // Toggle
```
just to make this more readable

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm
Comment (Sirryan2002) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm:68]:
```suggestion
	gorilla.visible_message("<span class='notice'>[gorilla] [gorilla.is_bipedal ? "stands up menacingly." : "drops back to all fours."]</span>",
		"<span class='notice'>You [gorilla.is_bipedal ? "stand up" : "get down on all fours."]</span>", 
		"<span class='notice'>You hear the sound of a gorilla rustling.</span>")
```
multiline for readability

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm
Comment (Sirryan2002) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm:208]: could you move this below the inherited vars please

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm
Comment (Henri215) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm:3]: What do you think about saying that they are from earth instead of central africa? Sounds too specific since only humans are from this planet.

Comment (S34NW) [reply code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm:3]: Sure

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm
Comment (Henri215) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm:3]:
```suggestion
	desc = "A ground-dwelling, predominantly herbivorous ape that inhabits the forests of central Africa on Earth."
```

File: code/modules/reagents/chemistry/reagents/misc_reagents.dm
Comment (lewcc) [inline code/modules/reagents/chemistry/reagents/misc_reagents.dm:714]: This should probably be a pretty noticeable message, maybe even more noticeable
```suggestion
		to_chat(M, "<span class='userdanger'>You feel your muscles swell and your hair grow as you return to monke.</span>")
```

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm
Comment (lewcc) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm:3]:
```suggestion
	desc = "A ground-dwelling, predominantly herbivorous ape that inhabits the forests of central Africa on Earth."
```

**Comment by Samman166** [comment]
Are the cargorillas player controlled? Or just NPCs

**Comment by S34NW** [comment]
> How does the cargoilla sprite look with the wooden crate type? the ones the tanks, floodlights and live animals use? Im slightly worried that it might look janky.

The big crate can not be carried by gorillas

> Are the cargorillas player controlled? Or just NPCs

They can be player controlled, in the same way that Betsy, Clucky, etc. can be

**Comment by ATP-Engineer** [comment]
I like the idea of 5 TC serum having opportunity as a last resort to a regular loadout. OR... Two serums and mindslave implant.

**Comment by SynthTwo** [comment]
God I am so excited for gorillas

**Comment by Sirryan2002** [comment]
waiting on TM for additional reviews / approval

File: code/modules/reagents/reagent_containers/hypospray.dm
Comment (GDNgit) [inline code/modules/reagents/reagent_containers/hypospray.dm:195]: this doesn't exist in your current branch, causing the injector to be invisible in game

**Comment by Zorazi-Teo-Daedalus** [comment]
Quick question. Is it intended that you're unable to put pet collars on gorillas? Or at least the Cargorilla. I tried because one went and got itself an ID, so I was gonna affix it to a pet collar and put it on them but was unable to attempt to put the collar on them like you can with some other mobs.

File: code/modules/reagents/reagent_containers/hypospray.dm
Comment (S34NW) [reply code/modules/reagents/reagent_containers/hypospray.dm]: Thanks, i deleted this to fix the issue.

**Comment by S34NW** [comment]
> Quick question. Is it intended that you're unable to put pet collars on gorillas? Or at least the Cargorilla. I tried because one went and got itself an ID, so I was gonna affix it to a pet collar and put it on them but was unable to attempt to put the collar on them like you can with some other mobs.

Not in my current plans, but someone could add it after. I don't want to further delay the PR.
I fixed access though, cargorillas should have cargo access.

**Comment by S34NW** [comment]
Latest commit:
Fixed cargorilla access.
Removed magillitis reagent and made it a bio-chip to prevent people murdering their targets via gorilla juice injector.
Buffed rampaging gorilla.
Swapped paralyse out for 40 stam loss on gorilla hit.

**Comment by Venuska1117** [comment]
Cargorillas cannot have name changed when they got equip pet collar (they have slot for it)

And people want be able to pick up people under hand and carry them around like crates (rotated 90 human sprite here)
and... some wanted be able to pick coffin and be coffin taxi gorilla (with killed me inside)

**Comment by SteelSlayer** [comment]
I have a report from somebody on discord that the radiation storm event turned _dead_ bodies in the morgue into gorillas. Have not confirmed this myself however.

**Comment by CinnamonSnowball** [comment]
> I have a report from somebody on discord that the radiation storm event turned _dead_ bodies in the morgue into gorillas. Have not confirmed this myself however.

I'm fairly sure I've seen it before, it's more of a problem with bodies in morgue trays being affected by radiation in general

**Comment by Coolrune206** [comment]
After seeing these TM'd on live for a bit, I can't say I'm a fan of the frequency of Gorillas appearing. Radiation storms turn people into Gorillas too frequently, and Genetics seems to be turning themselves into Gorillas somehow every fuckin' round.

I feel Gorillas should be significantly rarer, and crewmembers probably should never be turned into them at all, else I don't think this will be particularly beneficial to Paradise.

File: sound/creatures/gorilla.ogg
Comment (Sirryan2002) [inline sound/creatures/gorilla.ogg:1]: this is really loud in-game

**Comment by S34NW** [comment]
> After seeing these TM'd on live for a bit, I can't say I'm a fan of the frequency of Gorillas appearing. Radiation storms turn people into Gorillas too frequently, and Genetics seems to be turning themselves into Gorillas somehow every fuckin' round.
> 
> I feel Gorillas should be significantly rarer, and crewmembers probably should never be turned into them at all, else I don't think this will be particularly beneficial to Paradise.

>I have a report from somebody on discord that the radiation storm event turned dead bodies in the morgue into gorillas. Have not confirmed this myself however.

I dropped the prob from 1% to 0.1%, hopefully this will feel more organic. I've also stopped dead mobs from being gorrila'd

File: sound/creatures/gorilla.ogg
Comment (S34NW) [reply sound/creatures/gorilla.ogg:1]: lowered

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm
Comment (CinnamonSnowball) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm:2]:
```suggestion
	name = "gorilla"
```

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm
Comment (CinnamonSnowball) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm:207]:
```suggestion
	name = "cargorilla" // Overriden, normally
```

**Review by CinnamonSnowball — COMMENTED** [review]
Capitalized names!! Gnarg!!

**Review by SteelSlayer — APPROVED** [review]

**Comment by tipoima** [comment]
Does the chip bundle get the new chip?

**Pull request merged**
